README.md

```diff
@@ -65,6 +65,8 @@ custom hint metadata.
   If exceeded, 'et al' will be appended after this number.
 - `RECAPTCHA_SITE_KEY`
 - `RECAPTCHA_SECRET_KEY`
+- `TIMDEX_URL`: The GraphQL endpoint for Timdex/ArchiveSpace
+- `TIMDEX_SEARCH_URI`: the base search URL for the Timdex/ArchiveSpace application
```

Comment:
We probably want to call this `ASPACE_SEARCH_URI` or something similar as it may end up really confusing if we use TIMDEX for other content later on.

Reply:
Yeah, good call. I'm changing this in a forthcoming commit.